mods/ra/rules/world.yaml

```diff
@@ -1,5 +1,6 @@
 ^BaseWorld:
 	Inherits: ^Palettes
+	Inherits@MapGenerators: ^MapGenerators
```

Comment:
Maybe add it lower to the EditorWorld. There's no need to load all of this for the simulation world